src/EditorFeatures/Core/Implementation/MetadataAsSource/MetadataAsSourceFileService.cs

```diff
@@ -159,6 +181,87 @@ private string GetRootPathWithGuid_NoLock()
             return new MetadataAsSourceFile(fileInfo.TemporaryFilePath, navigateLocation, documentName, documentTooltip);
         }
 
+        private async Task<Document> DecompileSymbolAsync(Document temporaryDocument, ISymbol symbol, CancellationToken cancellationToken)
+        {
+            // Get the name of the type the symbol is in:
+            var containingOrThis = symbol.GetContainingTypeOrThis();
+            var total = GetFullReflectionName(containingOrThis);
```

Comment:
Nitpick: "reflectionQualifiedName" or something?

Reply:
:arrow_right: Fixed in 3a53d556e819164f8bf04409347f77b89e25f8d7